dgraph/cmd/graphql/schema/schema.go

```diff
@@ -37,30 +36,32 @@ type schRule struct {
 
 var schRules []schRule
 
-const (
-	INT      SupportedScalars = "Int"
-	FLOAT    SupportedScalars = "Float"
-	STRING   SupportedScalars = "String"
-	DATETIME SupportedScalars = "DateTime"
-	ID       SupportedScalars = "ID"
-	BOOLEAN  SupportedScalars = "Boolean"
-)
+type scalar struct {
+	name       string
+	dgraphType string
+}
+
+var supportedScalars = []scalar{
+	{name: "ID", dgraphType: "uid"},
+	{name: "Boolean", dgraphType: "bool"},
+	{name: "Int", dgraphType: "int"},
+	{name: "Float", dgraphType: "float"},
+	{name: "String", dgraphType: "string"},
+	{name: "DateTime", dgraphType: "dateTime"},
+}
 
 // AddScalars adds all the supported scalars in the schema.
 func AddScalars(doc *ast.SchemaDocument) {
-	addScalarInSchema(INT, doc)
-	addScalarInSchema(FLOAT, doc)
-	addScalarInSchema(ID, doc)
-	addScalarInSchema(DATETIME, doc)
-	addScalarInSchema(STRING, doc)
-	addScalarInSchema(BOOLEAN, doc)
+	for _, s := range supportedScalars {
+		addScalar(s, doc)
+	}
 }
 
-func addScalarInSchema(sType SupportedScalars, doc *ast.SchemaDocument) {
+func addScalar(s scalar, doc *ast.SchemaDocument) {
```

Comment:
All that this function does is do an append operation. I am not sure it deserves to be a function unless your are using it somewhere else?